### PR TITLE
[docs] Add note regarding cpp dev

### PR DESCRIPTION
**Pull request by cyber-toad**
https://github.com/orgs/organicmaps/discussions/5138

### CODE REVIEW

**Comment by strump** [comment]
@cyber-toad Thanks for PR. Could you please add links to Qt Creator official page (https://www.qt.io/product/development-tools), CLion (https://www.jetbrains.com/clion/) and all mentioned tools.

**Comment by cyber-toad** [comment]
@strump links are added, please take a look

File: docs/INSTALL.md [hunk 1]
```diff
@@ -345,6 +345,8 @@ In Android Studio open the project in `android/` directory.
 
 Setup a virtual device to use [emulator](https://developer.android.com/studio/run/emulator) ("Tools > Device Manager") or [use a hardware device for debugging](https://developer.android.com/studio/run/device). If using an emulator, make sure to choose a system image with API Level 31 and ABI _x86_64_.
 
+Android Studio has issues in parsing C++ part of the project - one of these applications should be used: [Qt Creator](https://www.qt.io/product/development-tools), [Visual Studio](https://visualstudio.microsoft.com/), [XCode](https://developer.apple.com/xcode/), [CLion](https://www.jetbrains.com/clion/).
```
Comment (biodranik) [inline docs/INSTALL.md:348]: 1. We didn't test if Visual Studio works, as OM is not compiled yet on Windows. Did you test it?
2. CLion and Qt Creator can open CMakeLists.txt files directly, while others need a manual cmake run with a specified generator to create an appropriate project file.

Comment (cyber-toad) [reply docs/INSTALL.md:348]: 1. No, I just took it from your comment in discussion, I can remove it if no one use it for now.
2. Should it be mentioned in my change?

Comment (biodranik) [reply docs/INSTALL.md:348]: You can leave only XCode and mention that project files can be generated with `cmake . -g Xcode`

Comment (cyber-toad) [reply docs/INSTALL.md]: Added a note about import for QT/Clion and how to generate files for XCode.

**Review by biodranik — APPROVED** [review]
Thank you!

**Pull request merged**